README.md

```diff
@@ -345,7 +345,7 @@ so it is a good idea to first create a Python `virtualenv`:
 ```
 $ virtualenv -p python3.8 venv
 $ . ./venv/bin/activate
-$ pip install pyspark
+(venv) $ pip install pyspark
```

Comment:
do we want to have this in requirements.txt?  Only do this if we anticipate having more dependancies in future